docs/source/basics.rst

```diff
@@ -0,0 +1,3 @@
+**********
+The Basics
```

Comment:
mypy's cheat sheet (https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html) is the most visited page in mypy's documentation, I'm assuming this will be something similar?